library/src/main/java/io/ona/kujaku/views/KujakuMapView.java

```diff
@@ -1059,7 +1026,7 @@ public void onResume() {
         getMapboxMap();
         // This prevents an overlay issue the first time when requesting for permissions
         if (warmGps && Permissions.check(getContext(), Manifest.permission.ACCESS_FINE_LOCATION)) {
-            checkLocationSettingsAndStartLocationServices(true);
+            checkLocationSettingsAndStartLocationServices(true, null);
```

Comment:
This does handle cases where location were disabled and user accepted to enable them.

Reply:
Sorry, I don't understand what this means, could you rephrase?

Reply:
This does not have any call back so does this matter anyway?

Reply:
Same as above.